packages/contracts/bin/deploy.ts

```diff
@@ -36,7 +35,7 @@ const main = async () => {
     sccSequencerPublishWindow: config.sccFraudProofWindow,
     ovmSequencerAddress: sequencer.address,
     ovmProposerAddress: sequencer.address,
-    ovmRelayerAddress: relayer.address,
```

Comment:
Note that this change reintroduces a bug, the bug being that both the sequencer and the relayer were using the same PK, namely the sequencer PK. This introduces the infamous nonce error, which is intermittent. The fix to the nonce error is to have the sequencer and the messenger use different addresses, which was fixed by correcting the bug (sequencer.address) to the correct value (relayer.address).

Reply:
Aha! Okay. That makes sense. Should we push this upstream too?

Reply:
this private key is set as the relayer private key in the relayer service (private key number three) where as optimism uses:
```
Account #8: 0x23618e81e3f5cdf7f54c3d65f7fbc0abf5b21e8f (10000 ETH)
l1_chain_1             | Private Key: 0xdbda1821b80551c9d65939329250298aa3472ba22feea921c0cf5d620ea67b97
```
https://github.com/ethereum-optimism/optimism/blob/develop/ops/docker-compose.yml#L110


not sure why this was changed!

Reply:
I think we changed it while making the changes to the CDM? do we need this change of account?

Reply:
I would presume we don't. But if we do - lets go hand in hand with the original repo. For now I'll set it to what Optimism did.